pkg/controller/cronjob/cronjob_controller.go

```diff
@@ -263,9 +264,8 @@ func syncOne(sj *batchv2alpha1.CronJob, js []batchv1.Job, now time.Time, jc jobC
 		glog.V(4).Infof("No unmet start times for %s", nameForLog)
 		return
 	}
-	if len(times) > 1 {
-		glog.V(4).Infof("Multiple unmet start times for %s so only starting last one", nameForLog)
-	}
```

Comment:
Please change this back, we want to log only when more than one time was left behind.